doc/Extensions/Applications.md

```diff
@@ -168,6 +169,17 @@ extend memcached /usr/local/bin/memcached
 4. Restart snmpd on your host
 5. On the device page in Librenms, edit your host and check `Memcached` under the Applications tab.
 
+### Proxmox
+1. Copy the [proxmox script](https://github.com/librenms/librenms-agent/blob/master/agent-local/proxmox) to `/usr/local/bin` (or any other suitable location) on your remote server.
+2. Make the script executable: `chmod +x /usr/local/memcached`
+3. Edit your snmpd.conf file (usually `/etc/snmp/snmpd.conf`) and add:
+```
+extend proxmox /usr/local/bin/proxmox
+```
+(Note: if your snmpd doesn't run as root, you might have to invoke the script using sudo. `extend proxmox /usr/bin/sudo /usr/local/bin/proxmox`)
+4. Restart snmpd on your host
+5. On the device page in Librenms, edit your host and check `Proxmox` under the Applications tab.
```

Comment:
In the code you don't actually check that this is enabled.

If this is because you only want to do it if you don't have data from check_mk then maybe move it into the first `(isset($config['enable_proxmox']) && $config['enable_proxmox'] && !empty($agent_data['app'][$name])) {` and check if $proxmox is set and if not then call your new code.